lib/LTP/Install.pm

```diff
@@ -0,0 +1,318 @@
+    assert_script_run "find /opt/ltp -name '*.run-test' > ~/openposix-test-list-$tag";
+}
+
+sub install_from_repo {
+    my ($tag) = @_;
+    zypper_call('in qa_test_ltp', dumb_term => 1);
+    script_run 'rpm -q qa_test_ltp | tee /opt/ltp_version';
+    assert_script_run q(find /opt/ltp/testcases/bin/openposix/conformance/interfaces/ -name '*.run-test' > ~/openposix-test-list-) . $tag;
+}
+
+sub install_runtime_dependencies {
+    if (is_jeos) {
+        zypper_call('in --force-resolution gettext-runtime', dumb_term => 1);
+    }
+
+    my @deps = qw(
+      sysstat
+      iputils
+    );
+    zypper_call('-t in ' . join(' ', @deps), dumb_term => 1);
+
+    # kernel-default-extra are only for SLE (in WE)
+    # net-tools-deprecated are not available for SLE15
+    # ntfsprogs are for SLE in WE, openSUSE has it in default repository
+    my @maybe_deps = qw(
+      acl
+      apparmor-parser
+      apparmor-utils
+      audit
+      bc
+      binutils
+      dosfstools
+      evmctl
+      fuse-exfat
+      kernel-default-extra
+      net-tools
+      net-tools-deprecated
+      nfs-kernel-server
+      ntfsprogs
+      numactl
+      psmisc
+      quota
+      sssd-tools
+      sudo
+      tpm-tools
+      wget
+      xfsprogs
+    );
+    for my $dep (@maybe_deps) {
+        script_run('zypper -n -t in ' . $dep . ' | tee');
+    }
+}
+
+sub install_runtime_dependencies_network {
+    my @deps;
+    # utils
+    @deps = qw(
+      ethtool
+      iptables
+      psmisc
+      tcpdump
+    );
+    zypper_call('-t in ' . join(' ', @deps), dumb_term => 1);
+
+    # clients
+    @deps = qw(
+      dhcp-client
+      telnet
+    );
+    zypper_call('-t in ' . join(' ', @deps), dumb_term => 1);
+
+    # services
+    @deps = qw(
+      dhcp-server
+      dnsmasq
+      nfs-kernel-server
+      rpcbind
+      rsync
+      vsftpd
+    );
+    zypper_call('-t in ' . join(' ', @deps), dumb_term => 1);
+}
+
+sub setup_network {
+    my $content;
+
+    # pts in /etc/securetty
+    $content = '# ltp specific setup\npts/1\npts/2\npts/3\npts/4\npts/5\npts/6\npts/7\npts/8\npts/9\n';
+    assert_script_run("printf \"$content\" >> /etc/securetty");
+
+    # ftp
+    assert_script_run('sed -i \'s/^\s*\(root\)\s*$/# \1/\' /etc/ftpusers');
+
+    # getaddrinfo_01: missing hostname in /etc/hosts
+    assert_script_run('h=`hostname`; grep -q $h /etc/hosts || printf "# ltp\n127.0.0.1\t$h\n::1\t$h\n" >> /etc/hosts');
+
+    # boo#1017616: missing link to ping6 in iputils >= s20150815
+    assert_script_run('which ping6 >/dev/null 2>&1 || ln -s `which ping` /usr/local/bin/ping6');
+
+    # dhcpd
+    assert_script_run('touch /var/lib/dhcp/db/dhcpd.leases /var/lib/dhcp6/db/dhcpd6.leases');
+
+    # echo/echoes, getaddrinfo_01
+    assert_script_run('sed -i \'s/^\(hosts:\s+files\s\+dns$\)/\1 myhostname/\' /etc/nsswitch.conf');
+
+    # SLE12GA uses too many old style services
+    my $action = check_var('VERSION', '12') ? "enable" : "reenable";
```

Comment:
As the 12GA is not supported anymore, this line could be dropped and the code cleaned a little bit?

Reply:
@baierjan good point, sent as separate PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7818